CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+2.4.1 (hold) - June 13, 2018
```

Comment:
Typically repo owners will do publishing and versioning; as a feature creator you don't need to edit CHANGELOG or equivalent files, so I'd remove them from the PR.